packages/core/src/RadioCheckToggleCard/RadioCheckToggleCard.stories.tsx

```diff
@@ -0,0 +1,1013 @@
+        const index = result.indexOf(value)
+
+        if (index >= 0) {
+          result.splice(index, 1)
+        } else {
+          result.push(value)
+        }
+      }
+
+      setExamples((previousState) => ({
+        ...previousState,
+        [name]: result,
+      }))
+    }
+
+  return (
+    <ThemeProvider>
+      <Section heading='Types'>
+        <Text textStyle='subheading3' my={3}>
+          Label
+        </Text>
+        <Text textStyle='paragraph' my={3}>
+          In its checked state, by default the color is to be primary.base.
+        </Text>
+        <Flex flexDirection='column' style={{ width: '50%', gap: '16px' }}>
+          <RadioCheckToggleCard
+            cardType='radio'
+            name='types-label'
+            title='Radio'
+            value='example-1'
+            isSelected={examples['types-label'].includes('example-1')}
+            onChange={handleOnChangeInLabel}
+          />
+          <RadioCheckToggleCard
+            cardType='checkbox'
+            name='types-label'
+            title='Checkbox'
+            value='example-2'
+            isSelected={examples['types-label'].includes('example-2')}
+            onChange={handleOnChangeInLabel}
+          />
+          <RadioCheckToggleCard
+            cardType='toggle'
+            name='types-label'
+            title='Toggle'
+            value='example-3'
+            isSelected={examples['types-label'].includes('example-3')}
+            onChange={handleOnChangeInLabel}
+          />
+        </Flex>
+      </Section>
+
+      {divider()}
+
+      <Section heading='States'>
+        <Text textStyle='paragraph' my={3}>
+          Radio - Examples shown indicate the card & action states
+        </Text>
+
+        <Flex my={4} flexWrap='wrap' style={{ width: '100%', gap: '16px' }}>
+          <EnabledStateCard style={{ flex: '1 1 30%' }}>
+            <RadioCheckToggleCard
+              cardType='radio'
+              hPosition='right'
+              vPosition='top'
+              name='states-radio'
+              title='Enabled'
+              value='example-1'
+              isSelected={false}
+              onChange={() => {}}
+            >
+              <Box width='100%' height='75px' />
+            </RadioCheckToggleCard>
+          </EnabledStateCard>
+
+          <HoverStateCard style={{ flex: '1 1 30%' }}>
+            <RadioCheckToggleCard
+              cardType='radio'
+              hPosition='right'
+              vPosition='top'
+              name='states-radio'
+              title='Hover'
+              value='example-2'
+              isSelected={false}
+              onChange={() => {}}
+            >
+              <Box width='100%' height='75px' />
+            </RadioCheckToggleCard>
+          </HoverStateCard>
+
+          <FocusStateCard style={{ flex: '1 1 30%' }}>
+            <RadioCheckToggleCard
+              cardType='radio'
+              hPosition='right'
+              vPosition='top'
```

Comment:
You can omit these three props everywhere since they're the same as the defaults for `RadioCheckToggleCard`